tests/e2e/sentinel/utils.js

```diff
@@ -57,6 +93,18 @@ const requestOnSentinelTestDb = (options) => {
   return utils.request(options);
 };
 
+//for e2e tests infodocs.js
+const requestOnSentinelTestDbNative = async (options) => {
+  if (typeof options === 'string') {
+    options = {
+      path: options,
+    };
+  }
+  options.path = '/' + constants.DB_NAME + '-sentinel' + (options.path || '');
+  return await utils.requestNative(options);
```

Comment:
Do you need to await the promise before returning or can the caller of `requestOnSentinelTestDbNative ` do the waiting?